src/subscription/data/details/sagas.js

```diff
@@ -0,0 +1,177 @@
+import {
+  call, put, select,
+} from 'redux-saga/effects';
+import { stopSubmit } from 'redux-form';
+import { camelCaseObject, convertKeyNames } from '../../../payment/data/utils';
+
+// Actions
+import {
+  subscriptionDetailsReceived,
+  subscriptionDetailsProcessing,
+  fetchSubscriptionDetails,
+  submitPayment,
+} from './actions';
+
+// Sagas
+import { handleErrors, handleMessages, clearMessages } from '../../../feedback';
+
+// Services
+import * as SubscriptionApiService from '../service';
+import { checkoutWithToken } from '../../../payment/payment-methods/cybersource';
+import { checkout as checkoutPaypal } from '../../../payment/payment-methods/paypal';
+import { checkout as checkoutApplePay } from '../../../payment/payment-methods/apple-pay';
+import { checkout as checkoutStripe } from '../../../payment/payment-methods/stripe';
+
+export const paymentMethods = {
```

Comment:
We will only have Stripe as payment method

Reply:
Done. 
```js

export const paymentMethods = {
  stripe: subscriptionStripeCheckout,
};
 
```